src/api-helper/machine/AbstractSoftwareDeliveryMachine.ts

```diff
@@ -324,3 +334,39 @@ export abstract class AbstractSoftwareDeliveryMachine<O extends SoftwareDelivery
     }
 
 }
+
+function toCodeInspectionCommand<PARAMS>(
+    eir: EnforceableProjectInvariantRegistration<PARAMS>): CodeInspectionRegistration<InvarianceAssessment, PARAMS> {
+    return {
+        name: `verify-${eir.name}`,
+        intent: !!eir.intent ? toStringArray(eir.intent).map(i => `verify ${i}`) : `verify ${eir.name}`,
+        parameters: eir.parameters,
+        projectTest: eir.projectTest,
+        onInspectionResults: eir.onInspectionResults,
+        description: eir.description,
+        tags: eir.tags,
+        targets: eir.targets,
+        repoFinder: eir.repoFinder,
+        repoFilter: eir.repoFilter,
+        repoLoader: eir.repoLoader,
+        inspection: eir.inspection || transformToInspection(eir.transform),
+    };
+}
+
+/**
+ * Return a CodeInspection that runs a transform and sees whether or it made
+ * an edit
+ * @param {CodeTransformOrTransforms<PARAMS>} transform
+ * @return {CodeInspection<InvarianceAssessment, PARAMS>}
+ */
+function transformToInspection<PARAMS>(transform: CodeTransformOrTransforms<PARAMS>): CodeInspection<InvarianceAssessment, PARAMS> {
+    const editor = toScalarProjectEditor(transform);
+    return async (p, i) => {
+        const result = await editor(p, i.context, i.parameters);
+        return {
+            id: p.id as RemoteRepoRef,
+            holds: result.edited,
```

Comment:
this sounds backwards ? The invariant holds if it was _not_ edited, eh?

Reply:
Yes. Thank you. Fixed.